src/Engine/Component/RigidBody.cpp

```diff
@@ -40,24 +39,38 @@ namespace Component {
         }
     }
 
-    glm::vec3 RigidBody::Position() const {
-        btTransform trans;
-        rigidBody->getMotionState()->getWorldTransform(trans);
+    glm::vec3 RigidBody::GetPosition() const {
+        btTransform trans = rigidBody->getWorldTransform();
         return Physics::btToGlm(trans.getOrigin());
     }
 
-    void RigidBody::Position(const glm::vec3& pos) {
-        btTransform trans;
-        rigidBody->getMotionState()->getWorldTransform(trans);
+    void RigidBody::SetPosition(const glm::vec3& pos) {
+        btTransform trans = rigidBody->getWorldTransform();
         trans.setOrigin(Physics::glmToBt(pos));
         rigidBody->setWorldTransform(trans);
     }
 
-    void RigidBody::Mass(float mass) {
+    glm::quat RigidBody::GetOrientation() const {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'GetOrientation' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)